# This file is the only file that should be changed when using this pacakage. 
# Change parameters to your prefrence

# MAP PARAMETERS -----------------------------------------------------------------------------------------------------------------------------------
MAP_SIZE: 500 # The map will be of size (MAP_SIZE,MAP_SIZE)
SCALE : 5 # SCALE OF MAP -- Resolution of map will be 1/SCALE
#---------------------------------------------------------------------------------------------------------------------------------------------------

# TOPICS -------------------------------------------------------------------------------------------------------------------------------------------
LASER_SCAN_TOPIC : "/manta/sonar" # Input topic on sensor_msgs/LaserScan.msg
ODOMETRY_TOPIC : "/odometry/filtered" # Input topic on nav_msgs/Odometry.msg format
MAP_PUBLISHING_TOPIC : "/map" # Topic the map is published on in nav_msgs/OccupancyGrid.msg format
OBJECT_PLACEMENT_TOPIC : "/objectPlacement" # Topic where object placement is published in vortex_msgs/ObjectPlacement.msg format
DARKNET_BBOX_TOPIC : "/darknet_ros/bounding_boxes" # Topic where bounding boxes are publsihed in darknet_ros_msgs/BoundingBoxes.msg format
#---------------------------------------------------------------------------------------------------------------------------------------------------

# OTHER --------------------------------------------------------------------------------------------------------------------------------------------
PUBLISHING_RATE : 0.5 # Rate map is published in
#---------------------------------------------------------------------------------------------------------------------------------------------------
